client/src/EntriesList.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react';
+import { Entry, readEntries } from './data';
+import { Link } from 'react-router-dom';
+import { FaPencilAlt } from 'react-icons/fa';
+
+export function EntriesList() {
+  const [entries, setEntries] = useState<Entry[]>([]);
+
+  useEffect(() => {
+    async function read() {
+      try {
+        const response = await readEntries();
+        setEntries(response);
+      } catch (err) {
+        console.error('Error:', err);
```

Comment:
You should display an error message to the user. An alert would be fine, or replacing the page with an error message like we did in previous exercises would also work.